### DISCUSSION TITLE
Is there a way to push new user code without downtime?

### DISCUSSION THREAD

**Discussion by pablo-statsig** [dagster-io/dagster#17544] [Q&A]
What is the best practice around safely pushing new code to code deployments? We use the graphql api to programmatically\ trigger jobs / check on their status however when we push new code and restart the code deplyments there is some downtime that causes these requests to fail. Is there some setting / way to safely push new code here that we are missing?

**Comment by shalabhc** [comment]
There should be no downtime when you push new code. Can you share more about which graphql request fails?
